### ISSUE TITLE
Great idea

### ISSUE THREAD

**Issue by paulca99** [JJ/github-pr-contains-action#56]
I'm having a bit of trouble running it though,  I have 2 jobs on my PR action, your action seems to work OK , but always fails at the end  with "Error: Not Found" , is it because I have a 2nd job  ?

```
name: 'Terraform GitHub Actions'
on:
  - pull_request
jobs:
  check_pr:
    runs-on: ubuntu-latest
    steps:
    - name: Check PR
      uses: JJ/github-pr-contains-action@releases/v0
      with:
        github-token: ${{github.token}}
        bodyContains: '[x]'
        bodyDoesNotContain: 'Delete this'
        diffContains: ''
        filesChanged: 1
        linesChanged: 1
  terraform:
    name: 'Terraform'
    runs-on: ubuntu-latest
    steps:
      - name: 'Checkout'
        uses: actions/checkout@master
      - name: 'Terraform Format'
        uses: hashicorp/terraform-github-actions@master
        with:
          tf_actions_version: 1.0.0
          tf_actions_subcommand: 'fmt'

        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

![image](https://user-images.githubusercontent.com/19557446/128158077-7ad27ba1-4aa1-4dff-acf3-67565eea3544.png)

**Comment by JJ** [comment]
Sorry, I missed this the first time. Let me check this and get back to you. Sorry for the delay.

**Comment by amitasinha1** [comment]
any resolution on this? I'm having the same issue, only using bodyDoesNotContain and getting Error: Not found

**Comment by marutinandanpandya** [comment]
+1 Facing the same issuu
<img width="545" alt="Screenshot 2022-08-09 at 6 10 53 PM" src="https://user-images.githubusercontent.com/4738021/183649372-5ef168b6-e904-4475-8e5d-70e748904e24.png">

**Comment by JJ** [comment]
Just back from the holidays, will have to take a look. The fact that someone else is actually using this was rather unexpected

**Comment by JJ** [comment]
@marutinandanpandya is it the same environment for you? Also two independent jobs?

**Comment by JJ** [comment]
Sorry for being so extraordinarily late @paulca99 could you please point me to the repo and action, so that I can see a bit more of the context?

**Comment by JJ** [comment]
Also, @paulca99, could you please check with v4? It's possibly fixed already in that version; IIRC, it failed if some key was missing.

**Comment by amitasinha1** [comment]
<img width="471" alt="Screen Shot 2022-08-15 at 7 11 16 PM" src="https://user-images.githubusercontent.com/83046859/184783676-b4995c75-4b5c-4f0a-8514-d02375a22ce8.png">
this is what i have and then creating a pr that only contains first_name, but still getting error:not found

**Comment by JJ** [comment]
I would say it's because of the empty strings in bodyContains and diffContains. I've produced a new version that should take care of diffContains, can you please check? Or, if it's a public repo, can I fork it and try to work it out?

**Comment by amitasinha1** [comment]
@JJ YESSS got it to work, amazing! The only thing is, I need to use this for where the diff does not contain keywords and I realized this only works for diffContains. Is there any way you could add diffDoesNotContain? I hope it's not too heavy of a lift

**Comment by amitasinha1** [comment]
@JJ also if i wanted to do multiple strings, can it take an array or no?

**Comment by JJ** [comment]
@amitasinha1 that's requested in #59, and the second in #32.
I'll check about the private repo, I have no idea what it can be. It's possible that you need another level for the GitHub token. I'll try and check in a private repo to see what's the problem.

**Comment by amitasinha1** [comment]
@JJ did #59 end up getting implemented? I see it as open still. Regarding #32, I get an error when i try to make it an array
<img width="1440" alt="Screen Shot 2022-08-17 at 9 25 57 AM" src="https://user-images.githubusercontent.com/83046859/185192598-b55f800d-0b53-403b-bdb9-ad4cafc7d90c.png">

**Comment by JJ** [comment]
No, it wasn't, still open... I guess I'll have to get around to do it.

**Comment by amitasinha1** [comment]
@JJ i'm not too familiar with js but i can try updating it? which file is the code in so i can open a pr?

**Comment by JJ** [comment]
Pretty much around here

https://github.com/JJ/github-pr-contains-action/blob/master/src/main.ts#L21

The problem is that it would need some refactoring, checking if the value is a string or an array, and so on. PRs are very welcome, of course.

**Comment by paulbakkerbloom** [comment]
Hi @JJ, have you gotten round to fixing the issue? I still get the Error: Not found -- also working in a private repo

**Comment by paulbakkerbloom** [comment]
Awesome! Do you have any idea when you'll have the time to work on it?

**Comment by JJ** [comment]
Should be any time soon, maybe today. Before early next week, at any rate.

**Comment by paulbakkerbloom** [comment]
Awesome, good luck!

**Comment by JJ** [comment]
This specific thing is being addressed in #32 . I'll be doing a release pretty soon.

**Comment by JJ** [comment]
Can you please try latest v7, to see if that works for you? #32 is implemented, diffDoesNotContain coming up... Anyway it's addressed in #59.

**Comment by JJ** [comment]
Please check that it solves your problem, reopen (or open a separate issue) if it does not.
